test/hotspot/jtreg/runtime/cds/DeterministicDump.java

```diff
@@ -50,8 +50,13 @@ public static void main(String[] args) throws Exception {
     public static void doTest(boolean compressed) throws Exception {
         ArrayList<String> baseArgs = new ArrayList<>();
 
-        // Use the same heap size as make/Images.gmk
+        // Try to reduce indeterminism of GC heap sizing and evacuation.
         baseArgs.add("-Xmx128M");
+        baseArgs.add("-Xms128M");
+        baseArgs.add("-Xmn128M");
+        baseArgs.add("-XX:ParallelGCThreads=1");
+        baseArgs.add("-XX:-ResizeTLAB");
+        baseArgs.add("-XX:-ResizePLAB");
```

Comment:
I am guessing we do not want to `-XX:-UseTLAB` summarily, so we get at least the same sized LABs all the time?

Reply:
See above.